libsodium-sys/Cargo.toml

```diff
@@ -1,25 +1,25 @@
 [package]
 authors = ["dnaq"]
 build = "build.rs"
-description = "FFI binding to libsodium"
+description = "FFI binding to libsodium generated with bindgen"
 documentation = "https://sodiumoxide.github.io/sodiumoxide"
 keywords = ["libsodium", "NaCl", "crypto"]
 license = "MIT OR Apache-2.0"
 links = "sodium"
 name = "libsodium-sys"
 repository = "https://github.com/sodiumoxide/sodiumoxide.git"
 categories = ["cryptography", "api-bindings"]
-version = "0.0.16"
+version = "0.0.17"
 
 [badges]
 travis-ci = { repository = "dnaq/sodiumoxide" }
 
 [build-dependencies]
 pkg-config = "0.3"
+bindgen = { git = "https://github.com/cynecx/rust-bindgen", branch = "canonicalize_pointer_ty" }
```

Comment:
Waiting on https://github.com/rust-lang-nursery/rust-bindgen/pull/1311 so that we don't have to depend on a fork

Reply:
It seems that the follow up RP has been merged as well. Waiting for that version bump :)

Reply:
`bindgen@0.37.0` has been released.